rhel/dockerfile/lex.go

```diff
@@ -134,9 +141,17 @@ Read:
 			_, err = l.sb.WriteRune(r)
 		case r == l.escchar:
```

Comment:
![image](https://user-images.githubusercontent.com/1027041/146275415-9d6024b0-0da9-4ba9-b661-16c65931bcf5.png)
Every time I read this file